### DISCUSSION TITLE
Is Birdnet-PI able to detect multiple species captured simultaneously ?

### DISCUSSION THREAD

**Discussion by croisez** [mcguirepr89/BirdNET-Pi#547] [General]
I know that in the Android app version of Birdnet, it is the case.
How about Birdnet-PI ?
Is the classifier code different ?

**Comment by w0mbatian** [comment]
I would also like to know the answer to this as in my first few hours of operation two bird species were identified at exactly the same time. When I play the two recordings they are identical to my ears.
So perhaps the answer is yes???
Ian
![One bird as two](https://user-images.githubusercontent.com/113158830/190878782-4dd62cb6-a0ef-41f8-9a6a-b98ae5e1fac5.jpg)

**Reply by ehpersonal38** [reply]
[From @CaiusX ](https://github.com/mcguirepr89/BirdNET-Pi/discussions/69#discussioncomment-2831891)

> In terms of the model it splits the audio into 3s chunks, analyzes and returns a list of birds & detections, filters that by confidence & reports the final result to the database & saves the sound file. There is a possibility of having multiple species being reported in a single chunk.

Definitely have seen this happen on my setup too!
